packages/status-react/src/system/tag/tag.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+
+import { useNavigate } from 'react-router-dom'
+
+import { TinyChevronRightIcon } from '../../icons/tiny-chevron-right-icon'
+import { TinyCommunityIcon } from '../../icons/tiny-community-icon'
+import { TinyReplyIcon } from '../../icons/tiny-reply-icon'
+import { styled } from '../../styles/config'
+import { Avatar } from '../avatar'
+import { Text } from '../text'
+
+const Base = styled('div', {
+  padding: '0px 6px',
+  border: '1px solid rgba(0, 0, 0, 0.1)',
+  borderRadius: '11px',
+  height: '22px',
+  display: 'flex',
+  alignItems: 'center',
+  width: 'max-content',
+  color: '$gray-1',
+  gap: '6px',
+  '&:hover': {
+    cursor: 'default',
+  },
+})
+
+const Segment = styled('div', {
+  display: 'flex',
+  alignItems: 'center',
+  gap: '4px',
+})
+
+const PathLink = styled('a', {
+  '&:hover': {
+    textDecoration: 'underline',
+  },
+})
+
+// fixme: clicking on flex gab/space between components captures and handles click events
+const Tag = (
+  props:
+    | {
+        type: 'community'
+        communityDisplayName: string
+        chatDisplayName: string
+        chatUuid: string
+        onNavigateChange: () => void
+      }
+    | { type: 'reply'; text: string }
+) => {
+  const { type } = props
+
+  // todo?: extract together with `PathLink`
+  const navigate = useNavigate()
+
+  switch (type) {
+    case 'community':
+      return (
+        <Base>
+          <Segment
+            role="none"
+            onClick={e => {
```

Comment:
You should be able to use `pointer-events: none` CSS property, instead of this.

Reply:
Doesn't work for me. Possible explanation below?

> the value none instructs the pointer event to go "through" the element and target whatever is "underneath" that element instead.
> ...
> In these circumstances, pointer events will trigger event listeners on this parent element as appropriate on their way to/from the descendant during the event capture/[bubble](https://developer.mozilla.org/en-US/docs/Web/API/Event/bubbles) phases.
>
>  – https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events